README.md

```diff
@@ -9,8 +9,48 @@ For technical details and help, go to [HACKING.md](HACKING.md)
 
 ## Running this site
 
+There are two primary ways to run this website locally. We recommend using the `./run` command, which runs the site through Docker. The site can also run directly on the system with our build tools and Polymer.
 
-### Dependencies
+
+### `./run` command
+
+
+#### Dependencies
+
+- Latest version of Docker
+
+
+#### Quick start
+
+Start up a local server which watches serves content from the `examples` folder:
+``` bash
+$ ./run
+```
+
+ Start up a local server which watches a given file or folder for changes relative to the project root.
+``` bash
+./run serve [file or folder]
+```
+
+
+#### Usage
+
+- `./run`: Run local server with example content.
+- `./run serve [file or folder]`: Run server and watch a local file or folder.
+- `./run help`: Print help.
+
+NPM/Yarn scripts:
+- `./run yarn serve-live`: Run local server with live content.
+- `./run yarn build-all`: Generate live tutorials and build live site.
+- `./run yarn build-site`: Build site to `build` folder.
+- `./run yarn build-tutorials`: Generate live tutorials.
+- `./run yarn polymer [args]`: Run a command through Polymer.
+
+
+### Running without Docker
```

Comment:
Generally I prefer READMEs to be short and unassuming, which is why [www.ubuntu.com's](https://github.com/canonical-websites/www.ubuntu.com/blob/master/README.md) just says "For more detailed local development instructions, see HACKING.md." Is this something you'd like to emulate?